js/src/lib/interfaces/Types.ts

```diff
@@ -38,6 +38,7 @@ export enum PipelineType {
 	"image-to-text"                                           = "Image-to-Text",
 	/// others
 	"structured-data-classification"                          = "Structured Data Classification",
+	"reinforcement-learning"				  = "Reinforcement Learning",
```

Comment:
can you use spaces instead of tabs? (same elsewhere)

Reply:
Sure!

Reply:
BTW this shouldn't take engineering time from @Pierrci IMO, you should check with library maintainers @LysandreJik @osanseviero @muellerzr when CI is red on your PR